sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -484,6 +485,7 @@ export interface BlobDeleteIfExistsResponse extends BlobDeleteResponse {
 // @public
 export interface BlobDeleteOptions extends CommonOptions {
     abortSignal?: AbortSignalLike;
+    blobDeleteType?: "permanent" | string;
```

Comment:
remove string ?

Reply:
This is only temporary. Will update when we fix generated code to use a string enum.